src/chrome/content/rules/worldofplayers.de.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Invalid Certificate:
+	acropolis.worldofplayers.de
+	dlord.worldofplayers.de
+	dragonage.worldofplayers.de
+	farcry.worldofplayers.de
+	forenarchiv.worldofplayers.de
+	lego.worldofplayers.de
+	messiah.worldofplayers.de
+	rpb.worldofplayers.de
+	witcher.worldofplayers.de
+	wogen.worldofplayers.de
+	wotor.worldofplayers.de
```

Comment:
Wildcard DNS record.

Reply:
They have no wildcard certificate. 

Or do you want a info like this: "Invalid Certificate - All other targets" ?

Reply:
You can just add a comment that they use a wildcard DNS record. That's just a hint that nobody starts to add random hosts to the list.

Reply:
But i can´t find there a wildcard dns record. All targets have their own certificates.

Reply:
See e.g. `foobar.worldofplayers.de`